profitbricks-rest/pom.xml

```diff
@@ -70,6 +70,13 @@
            <artifactId>jclouds-okhttp</artifactId>
            <version>${jclouds.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.5-SNAPSHOT</version>
+            <scope>test</scope>
+            <type>jar</type>
+        </dependency>
```

Comment:
This dependency is added only for a String join. Use the available methods in the Guava `Strings` class and remove this extra dependency.